src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -213,7 +213,7 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ dispatch, editData }) => {
   };
 
   return (
-    <Form onSubmit={handleSubmit(setWorld)} onChange={handleOnChange}>
+    <S.FormWrapper onSubmit={handleSubmit(setWorld)} onChange={handleOnChange}>
```

Comment:
Can just as easily add `<Form className="">` and deal with it in an SCSS file

Reply:
fixed in d205d698e7827ce16d8dfff434b02f7df4ff5fd1